test/verify/check-machines

```diff
@@ -2107,6 +2109,10 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         def createAndVerifyVirtInstallArgs(self):
             self.browser.click(".pf-c-modal-box__footer button:contains(Create)")
             self.browser.wait_not_present("#create-vm-dialog")
+            if self.storage_pool != "No storage":
```

Comment:
This does not belong here nicely, createAndVerifyVirtInstallArg as the title says, makes sure that the CLI args in the generated virt install command are correct.

Not a big deal, so i will merge it.